js/stack/stack-dom.js

```diff
@@ -20,18 +21,52 @@ renderListStack();
 
 const generateWarningStack = type => {
   if (type === 'underflow') {
-    // ... your code goes here
+    // console.log("The Stack is full")
+    warningBottomStack.style.display = "block"
+    warningBottomStack.innerHTML = 'underflow'
+
   } else if (type === 'overflow') {
-    // ... your code goes here
+    // console.log("The Stack is Empty")
+    warningTopStack.style.display = "block"
+    warningTopStack.innerHTML = 'underflow'
+  } else {
+    warningTopStack.style.display = "none"
+    warningBottomStack.style.display = "none"
   }
 };
 
 const addToStack = () => {
-  // ... your code goes here
+  // console.log("We are trying to add a new Stack") test
+  newStack.push(stackInput.value)
+  console.log(newStack.stackControl)
+  if (!newStack.canPush()) {
+         generateWarningStack('overflow')
+  } else {
+    warningTopStack.style.display = "none"
+  }
+ // console.log(newStack.stackControl.length)
+  const liEntryInactive = document.querySelector('.inactive')
+  liEntryInactive.classList.add('active')
+  liEntryInactive.classList.remove('inactive')
+  liEntryInactive.innerHTML = stackInput.value
```

Comment:
No sé si te has fijado pero el botón de add to stack añade elementos a la queue y viceversa.

Eso es porque has seleccionado todos los .inactive y no solo los del stack.
```suggestion
 // console.log(newStack.stackControl.length)
  const liEntryInactive = stackList.querySelector('.inactive')
  liEntryInactive.classList.add('active')
  liEntryInactive.classList.remove('inactive')
  liEntryInactive.innerHTML = stackInput.value
```

Reply:
Si lo he visto y por eso he hecho un tercera commit pero demasiado tarde jeje